heidrun/uva_mods.rb

```diff
@@ -0,0 +1,180 @@
+# coding: utf-8
+
+#   <accessCondition type="restrictionOnAccess"
+#                    displayLabel="Access to the Collection"/>;
+#   <accessCondition type="useAndReproduction"
+#                    displayLabel="Use of the Collection">
+rights_map = lambda do |record|
+  rights = record['mods:accessCondition']
+           .match_attribute(:type, 'restrictionOnAccess')
+           .match_attribute(:displayLabel, 'Access to the Collection')
+
+  rights.concat(record['mods:accessCondition']
```

Comment:
Regarding the use of `#concat` here and below, can you confirm that this should be concatenating a string? (If there are multiple rights statements, should they be concatenated into one string, with the assumption that there is an appropriate full stop plus space character in between sentences?  Is `record#match_attribute` ever going to return `nil`, such that `#concat` will result in a `NoMethodError`?

Just checking!  I may be unclear as to the type of `#match_attribute`.